pkgs/applications/science/electronics/fritzing/default.nix

```
@@ -1,41 +1,75 @@
-{ mkDerivation, stdenv, fetchpatch, fetchFromGitHub, qmake, pkgconfig
-, qtbase, qtsvg, qtserialport, boost, libgit2
+{ mkDerivation, stdenv, fetchgit, fetchFromGitHub, qmake, pkgconfig
+, qtbase, qtsvg, qttools, qtserialport, boost, libgit2_0_27
 }:
 
+let
+  # tag corresponding to a release, has no further relation
+  fritzingTag = "CD-498";
+  # SHA256 of the fritzing-parts HEAD on the master branch,
+  # which contains the latest stable parts definitions
+  partsSha = "e79a69765026f3fda8aab1b3e7a4952c28047a62";
+in
+
 mkDerivation rec {
   pname = "fritzing";
-  version = "0.9.3b";
+  version = "0.9.4";
```

Perhaps the `fritzingTag` should also be added to the `version`, because there multiple 0.9.4 releases, e.g., CD-415 and CD-498.
```suggestion
  version = "0.9.4-${fritzingTag}";
```

---

I would not want to add this, as it looks ugly to me and is just a (mostly) random number. If this ever happens again and make multiple releases for a version, we might simply append a rev number at the end.

Otherwise, we could just append the build number, without the `CD` prefix...?!

---

I totally feel you and also consider it to be unaesthetic. However, the project does not use semantic versioning and "0.9.4" is ambiguous. Just adding the build number is also fine for me. I just want to be able to distinguish versions.

---

OK, I changed the version to include the build number. I can squash the commits if wanted, or someone squashes when merging...

---

Squashing those into one commit sounds good, I think. Otherwise this PR looks good to me, thanks again (: